src/dash-table/syntax-tree/lexeme/logical.ts

```diff
@@ -0,0 +1,49 @@
+import Logger from 'core/Logger';
+import { LexemeType, IUnboundedLexeme } from 'core/syntax-tree/lexicon';
+
+enum LogicalOperator {
+    And = '&&',
+    Or = '||'
+}
+
+export const and: IUnboundedLexeme = {
+    evaluate: (target, tree) => {
+        Logger.trace('evaluate -> &&', target, tree);
+
+        const t = tree as any;
+        const lv = t.left.lexeme.evaluate(target, t.left);
+        const rv = t.right.lexeme.evaluate(target, t.right);
+        return lv && rv;
+    },
+    type: LexemeType.LogicalOperator,
+    priority: 2,
+    regexp: /^(and\s|&&)/i,
```

Comment:
probably want to allow `(` immediately after, like `(1<2)and(3<4)` - that works in Python anyway, will confuse ppl if it fails. Same for `or`.

Reply:
This is allowed in full query form -- e.g. `{a} eq "1" and ({a} eq "0" or ({b} eq "1" or {b} eq "0"))`

Reply:
Ah! You mean the space?